CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to pytket
+
+Pull requests are welcome. To make a PR, first fork the repo, make your proposed
+changes on the `main` branch, and open a PR from your fork. If it passes
+the checks and is accepted after review, it will be merged in.
```

Comment:
Maybe we should write something about the manual build? If there are any changes to the manual it needs to be rebuild.